### ISSUE TITLE
Hyperparams for CARS?

### ISSUE THREAD

**Issue by ifeherva** [MalongTech/research-ms-loss#16]
First of all thank you for sharing the code, this is really great work.

I am trying to reproduce the results on other datasets (CARS, SOP, In-Shop), both with resnet50 and inception-bn, could you please share the hyperparams that you used? I tried the default ones and a few tweaking but I could not get the numbers in the paper.

Thank you!

**Comment by jeromerony** [comment]
Hello,
I am not one of the author, but if you check the paper carefully, it is written in appendix that they used a batch size of 1000 for SOP and they do not report for InShop and Cars. So you might have trouble finding the answer to your questions.
Also, after contacting the authors by mail, they refused to share the code and hyper-parameters for the experiments.
Best.

**Comment by torki-hossein** [comment]
> First of all thank you for sharing the code, this is really great work.
> 
> I am trying to reproduce the results on other datasets (CARS, SOP, In-Shop), both with resnet50 and inception-bn, could you please share the hyperparams that you used? I tried the default ones and a few tweaking but I could not get the numbers in the paper.
> 
> Thank you!

Hello,
I'm not one of the author also but I think that all of parameters saved in cfg file and it exist now with code. s.t like "SCALE_NEG", "SCALE_POS", "Max_Iteration", "Gamma", "Momentum", "Warmup_factor" and etc exist in the cfg file.
best regards.

**Comment by Totoro-wen** [comment]
> > First of all thank you for sharing the code, this is really great work.
> > I am trying to reproduce the results on other datasets (CARS, SOP, In-Shop), both with resnet50 and inception-bn, could you please share the hyperparams that you used? I tried the default ones and a few tweaking but I could not get the numbers in the paper.
> > Thank you!
> 
> Hello, I'm not one of the author also but I think that all of parameters saved in cfg file and it exist now with code. s.t like "SCALE_NEG", "SCALE_POS", "Max_Iteration", "Gamma", "Momentum", "Warmup_factor" and etc exist in the cfg file. best regards.
hello,
I have the same problem, may I ask how do you adjust these hyper-parameters.
best

**Comment by Totoro-wen** [comment]
> > First of all thank you for sharing the code, this is really great work.
> > I am trying to reproduce the results on other datasets (CARS, SOP, In-Shop), both with resnet50 and inception-bn, could you please share the hyperparams that you used? I tried the default ones and a few tweaking but I could not get the numbers in the paper.
> > Thank you!
> 
> Hello, I'm not one of the author also but I think that all of parameters saved in cfg file and it exist now with code. s.t like "SCALE_NEG", "SCALE_POS", "Max_Iteration", "Gamma", "Momentum", "Warmup_factor" and etc exist in the cfg file. best regards.

hello,
I have the same problem, may I ask how do you adjust these hyper-parameters.
best
